richards/src/main/scala/org/scalajs/benchmark/richards/Richards.scala

```diff
@@ -45,12 +45,28 @@
 
 package org.scalajs.benchmark.richards
 
+import org.scalajs.benchmark.{Benchmark, BenchmarkSuite}
+import scalajs.js
+
+object RunRichards extends js.JSApp {
+  def main(): Unit = {
+    new RichardsSuite
+    BenchmarkSuite.runSuites()
+  }
+}
+
 /**
  * Richards simulates the task dispatcher of an operating system.
  */
-object Richards extends org.scalajs.benchmark.Benchmark {
+class RichardsSuite extends BenchmarkSuite {
```

Comment:
I recommend not to delete `object Richards`, but rather use `Richars.run` in `val benchmarks` herebelow. Just remove `extends org.scalajs.benchmark.Benchmark`, and move the `object Benchmarks {` line just before `def run()`.

The same should be done for the other benchmarks.